src/psyclone/tests/psyGen_test.py

```diff
@@ -2835,6 +2925,26 @@ def test_symbol_can_be_printed():
     assert "sname<real, [], local>" in str(symbol)
 
 
+def test_symbol_gen_c_definition():
+    '''Test that the Symbol gen_c_definition method generates the expected
+    C definitions, or raises an error if the type is not supported.
+    '''
+    s1 = Symbol("name", "integer", [])
+    assert "int name" == s1.gen_c_definition()
```

Comment:
pylint complains about the order of this comparison (and below).

Reply:
fixed